example/demo_ui.py

```diff
@@ -62,7 +62,7 @@ def genRANstring(n):
 
 # CREATE THE SIDEBAR
 with st.sidebar:
-    st.image('img/RMIT_POS3.png', use_column_width=True)
+    st.image('example/img/log.png', use_column_width=True)
```

Comment:
Is there any logo that related to your team?. I mean the logo without SkywardAI also be fine.